ChunkyDocs/docs/reference/technical/scene_format.md

```diff
@@ -0,0 +1,396 @@
+
+There are a few different Octrees that are available within Chunky being NODE (legacy), PACKED (default), and BIGPACKED all which have different pros and cons. Available Octrees can be expanded via [Plugins](../../../plugins/chunky_plugins) such as aTom3333's [Octree plugin](../../../plugins/plugin_list#octree-plugin).
+
+There are a few different BVH build methods available within Chunky being SAH_MA (default), SAH, and MIDPOINT all which have different pros and cons. Available BVHs can be expanded via [Plugins](../../../plugins/chunky_plugins) such as aTom3333's [BVH plugin](../../../plugins/plugin_list#bvh-plugin).
+
+### Octree Format
+
+`<version int> <block palette data> <world octree data> <water octree data> <grass tinting data> <foliage tinting data> <water tinting data if version >= 4>`
+
+<a href="https://github.com/chunky-dev/chunky/blob/master/chunky/src/java/se/llbit/chunky/resources/OctreeFileFormat.java#L50" target="_blank">The section of code</a>.
+
+
+#### Block Palette Data
+
+Stores NBT tags for each block.
+
+`<version int == 4> <number of block types> <Serialized NBT tags for each block in order>`
+
+
+#### World and water Octree Data
+
+"octree is pretty complex lol"
+
+"The octree itself is something like storing it depth first with 0xFFFF FFFF as a node and the type if it is a leaf."
+
+
+#### World/Tint Textures
+
+Stores tint colors for grass, foliage, and water as WorldTexture.
+
+`<number of tiles (chunks)> for each tile: <chunk x coordinate int> <chunk y coordinate int> <chunk texture in x major order, rgb as floats in linear color space>`
+
+---
+
+## Emittergrid
+
+The .emittergrid is only generated if Emitter Sampling Strategy is set to One or All. Is also GZIP compressed.
+
+* Version 0 - `<version as int> <grid size as int> <cell size as int> for each emitter position <positions as ints, -x, -y, -z corner, +0.5 to get center> for each grid <number of emitters in grid, index of emitters in positions array>`
+
+* Version 1 -`<version as int> <cell size as int> <grid offset x> <grid size x> <grid offset y> ...`
+
+* Version 2 - `<x,y,z center float, radius as float> for emitter position`
+
+---
+
+## Scene Description Format (SDF)
+
+Most of the settings in Chunky scenes are stored in Scene Description files using a JSON-based file format. This page documents the SDF file format. The documentation is currently incomplete, and may lag behind the current Chunky version as new versions are released. Check the version history at the end of this page to see the latest updates made to the SDF documentation.
+
+SDF JSON files are stored in the scene directory and the filename is based on the scene name with `.json` appended. For example, the JSON file for a scene named `MyScene` would be `MyScene.json`.
+
+
+### Structure
+
+#### General
```

Comment:
TODO - Update contents of SDF. ie `emitterSamplingStrategy | {“NONE”, “ONE”, “ALL”}` for 2.5.0.

Please defer this to a new issue as this won't block merging of this PR but this does need to be updated at some point.

Reply:
An issue has been created.